docs/pages/guides/sharing-before-shipping.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Sharing your app before you ship
+---
+
+Ultimately, you will want to publish your application to the App Store and Play Store to access the opportunities for wider distribution and to provide a familiar installation process for your users.  While you are developing though, you'll want to let other members of your team and your potential users see the work in progress to gather feedback and make sure you're building a quality product.
+
+## Expo Go
+
+If you are using the Expo Managed workflow, you can share your application with your team through the Expo Go app.
+
+1. Publish the latest version of your project to Expo's servers by running `expo publish`.
+2. [Invite your teammate](https://expo.io/[account]/[project]/settings/members) to the Expo account that owns the project.
+3. Have them [download the Expo Go app](https://expo.io/client) onto their device.
+4. They'll be able to open your application from the Profile tab of their Expo Go app.
+
+
+## Internal Distribution
+
+If you want to share outside of Expo Go, Android and iOS both offer ways to install a build of your application directly on devices.
+
+### on Android
+
+To share your application to Android devices, you must build an APK of your project (rather than an AAB that is optimized for distribution through the Play Store), but that APK can be downloaded (from the browser, email or chat app) and installed on any Android device once the user accepts the security risk of installing an app that has not gone through Play Store review.
```

Comment:
```suggestion
To share your application to Android devices, you must build an APK of your project (rather than an AAB that is optimized for distribution through the Play Store), but that APK can be downloaded (from the browser, email, or chat app) and installed on any Android device once the user accepts the security risk of installing an app that has not gone through Play Store review.
```

+ oxford morning

the security risk is a good point, albeit a little scary for people that overthink it